cli/src/command/project.rs

```diff
@@ -143,8 +156,56 @@ impl CommandT for Register {
         project_registered.result?;
         println!(
             "✓ Project {}.{} registered in block {}",
-            self.project_name, self.org_id, project_registered.block,
+            self.project_name, project_domain, project_registered.block,
         );
         Ok(())
     }
 }
+
+#[derive(Clone, Eq, PartialEq, Debug, serde::Deserialize, serde::Serialize)]
+#[serde(rename_all = "lowercase")]
+enum DomainType {
+    Org,
+    User,
+}
+
+impl FromStr for DomainType {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "org" => Ok(Self::Org),
+            "user" => Ok(Self::User),
+            _ => Err("Invalid DomainType".to_string()),
```

Comment:
This is the message that is displayed to the user when they use a wrong domain. To make the message more helpful it should avoid code terminology (`DomainType`), repeat the input string, and list the possibilities.
```suggestion
            input => Err(format!("Invalid domain type \"{}\". Must be either org or user.", value)),
```

Reply:
It actually isn't given that we define the possible values in the struct opt construct. This case should therefore never happen and the user should never see that error message.

Reply:
To be more exact, the error users get is:

```
error: 'ORG' isn't a valid value for '<domain-type>'
        [possible values: org, user]
```